presto-product-tests/src/main/java/io/prestosql/tests/hive/TestHiveStorageFormats.java

```diff
@@ -537,6 +540,117 @@ private void runTimestampQueries(String tableName, List<TimestampAndPrecision> d
         onPresto().executeQuery("DROP TABLE " + tableName);
     }
 
+    @Test(dataProvider = "storageFormatsWithNanosecondPrecision", groups = STORAGE_FORMATS)
```

Comment:
How much time does the test take?
How many times do we run the test on CI? 

storage formats are run in multiple configurations -- for good reason. Since HDFS access patterns vary between formats, we run these tests for all supported HDFS authentication/impersonation configurations. This one, however, doesn't need to be run for all of them. Thus --and especially if it takes significant amount of time -- we should change its group definiion. Perhaps, it would be enough to a) remove storage_format group and b) add a comment why the test is **not** in that group.

(this applies to existing test for nonstructural timestamp as well)

Reply:
This test doesn't always have the same behavior for different storage formats because of time zone storage differences.